.github/linters/.flake8

```diff
@@ -0,0 +1 @@
+extend-ignore = ABC123
```

Comment:
Is `ABC123` some actual flake8 error code?

Reply:
Actually forgot to remove those lines of code. We were referring to [https://stackoverflow.com/questions/73568036/flake8-line-break-before-binary-operator-how-to-fix-it](https://stackoverflow.com/questions/73568036/flake8-line-break-before-binary-operator-how-to-fix-it)

But fixed here [https://github.com/redhat-performance/opl/pull/65/files](https://github.com/redhat-performance/opl/pull/65)